applications/Chat/coati/trainer/strategies/ddp.py

```diff
@@ -98,18 +98,18 @@ def save_pretrained(
             pretrained_model.save_pretrained(path, save_function=lambda *args, **kwargs: None)
             if tokenizer is not None:
                 tokenizer.save_pretrained(path)
-        model_path = os.path.join(path, "pytorch_model.bin")
-        self.save_model(model, model_path, only_rank0=only_rank0)
+        # model_path = os.path.join(path, "pytorch_model.bin")
+        self.save_model(model, path, only_rank0=only_rank0)
 
-        def _replace_keys(model_path: str, replace_fn: Callable):
-            state_dict = torch.load(model_path, map_location="cpu")
-            state_dict = {replace_fn(k): v for k, v in state_dict.items()}
-            torch.save(state_dict, model_path)
+        # def _replace_keys(model_path: str, replace_fn: Callable):
```

Comment:
If theses codes are useless, then delete them

Reply:
This is useful for replacing the prefix of keys in a checkpoint.